README.md

```diff
@@ -1,73 +1,31 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# hello microverse
 
-> Description the project.
+> the porpose of this project is to test our knowledge on git and github flows.
 
 
```

Comment:
- [ ] Kindly use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please don't forget to edit all the sections to better suit your project.

Reply:
- Please add the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by microverse to this branch `feature-b` . You added the README to the `master` branch instead of `feature-b`. Please fix that.